modules/odc-editing-health-checks.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// applications/odc-monitoring-application-health-using-developer-perspective
+
+[id="odc-editing-health-checks"]
+= Editing health checks
+
+You can use the *Topology* view to edit health checks added to your application, modify them, or add more health checks.
+
+.Procedure
+. In the *Topology* view, right-click your application and select *Edit Health Checks*. Alternatively, in the side panel, click the *Actions* drop-down list and select *Edit Health Checks*.
+. In the *Edit Health Checks* page:
+
+* To remove a previously added health probe, use the minus sign adjoining it.
+* To edit the parameters of an existing probe:
++
+.. Click the *Edit Probe* link next to a previously added probe or click on the probe to see the parameters for the probe.
```

Comment:
click on -> click ?

Not sure about this one.

Reply:
Click, as you suggest, would be correct. But the developer asked me to remove that part, so have taken that part out.